samples/fabric/fabric-cli/scripts/ci-test.sh

```diff
@@ -0,0 +1,40 @@
+COUNT=0
```

Comment:
Where is this script getting called from? And should we add a note in the Fabric-CLI readme about configuring and running this?

Reply:
Oh sorry I didn't notice, this was a dummy script, wasn't supposed to be checked in (probably isn't debugged as well). I'll delete this file.